docs/TESTS.md

```diff
@@ -1,15 +1,114 @@
 # Tests
 
-<!-- TODO: write document
+This track uses the [unitt unit-testing framework][unitt].
 
-  This document should describe everything related to running tests in the track.
+Each exercise will have a similar structure as seen below:
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+```
+HELP.md
+README.md
+src
+tester.art
+tests
+```
 
-  This document can also link to the testing framework documentation.
+To run the exercise's test suite on the CLI, run `arturo tester.art` (or `exercism test`) inside the exercise folder.
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/tests`.
+## Structure
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+Your solution goes inside `src\<slug>.art` where `slug` is the exercise slug.
```

Comment:
The \ as a path separator took me a moment. I'm so used to Unix /

Reply:
l didn’t even realize I did that. Thanks.